hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/KafkaDataConnectionIntegrationTest.java

```diff
@@ -89,17 +91,32 @@ public void test_nonShared() {
     }
 
     @Test
-    public void test_shared() {
+    public void when_createSharedProducer_then_success() {
+        String dcName = randomName();
+        String name = createRandomTopic(PARTITION_COUNT);
+        createSqlKafkaDataConnection(dcName, true);
+        createKafkaMappingUsingDataConnection(name, dcName, constructMappingOptions("int", "varchar"));
+
+        try (SqlResult r = sqlService.execute("INSERT INTO " + name + " VALUES (0, 'value-0')")) {
+            assertThat(r.updateCount()).isZero();
+        }
+
+        assertTipOfStream("SELECT * FROM " + name, singletonList(new Row(0, "value-0")));
+    }
+
+    @Test
+    public void when_createSharedProducerWithOverriddenProperty_then_success() {
```

Comment:
Minor comment: shouldn't it be `when_createSharedProducerWithOverriddenProperty_then_fail()` ?